087-adopt-connect-health-endpoint.md

```diff
@@ -0,0 +1,84 @@
+# Adopt the Kafka Connect health check endpoint
+
+In a Kubernetes node, the kubelet component uses the configured liveness probe to know when to restart a container (i.e. deadlock).
+Instead, it uses the configured readiness probe to know when a container is ready to start accepting traffic (i.e. startup phase completed).
+A common pattern for liveness probes is to use the same HTTP endpoint as for readiness probes.
```

Comment:
I don't think that is a common pattern ... but if you have nothing better, it is an option.

Reply:
Getting rid of this as not important in this context.